libtelemetry/src/test/java/com/mapbox/android/telemetry/SchemaTest.java

```diff
@@ -56,6 +63,12 @@ public void checkAppUserTurnstileEventFields() throws Exception {
   @Test
   public void checkLocationEventSize() throws Exception {
     JsonObject schema = grabSchema(LOCATION);
+    if (schema != null) {
```

Comment:
Do we have an issue opened to circle back and modify this test to:
- Read through the `schema` array and ensure that all required fields are present
- Read through the `fields` array and ensure that no fields are missing from the `schema`

Reply:
https://github.com/mapbox/mobile-telemetry/issues/521